docker/ol-importbot-start.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+python --version
+scripts/manage-imports.py "$OL_CONFIG" $IMPORTBOT_OPTS 7100
```

Comment:
Mimic this: https://github.com/internetarchive/olsystem/blob/master/etc/supervisor/conf.d/import-bot.conf ; looks like we also need `import-all`; there is no port (so can drop it from here and the docker compose file).